src/portal/src/app/project/create-project/create-project.component.html

```diff
@@ -91,7 +91,7 @@ <h3 class="modal-title">{{'PROJECT.NEW_PROJECT' | translate}}</h3>
                 <label for="create_project_storage_limit" class="clr-control-label"></label>
                 <div class="clr-control-container">
                     <div class="clr-input-wrapper">
-                        <label class="clr-control-label endpoint">{{ 'PROJECT.ENDPOINT' | translate }}</label>
+                        <label class="clr-control-label endpoint required">{{ 'PROJECT.ENDPOINT' | translate }}</label>
```

Comment:
The required icon besides 'Endpoint' looks strange, because it is a readonly field. Could you add the required icon in proxy cache when it's enabled.

Reply:
I think adding it to "Proxy cache" means it's a must, but in fact it's optional. When proxy cache is enabled, you must choose a endpoint.  Endpoint is not a readonly field, it's a selection item. you can not input, but you must select one